app/js/components/filterComponents/actionsSelect/actionsSelect.directive.js

```diff
@@ -36,21 +37,31 @@ function actionsSelectCtrl($rootScope,
 
     $scope.select = function (option) {
         $scope.selected = option;
-        MultiButtonService.setState('inventoryWithActions', option.withActions);
-        MultiButtonService.setState('inventoryWithoutActions', option.withoutActions);
+        setSystemHealth(option);
         FilterService.doFilter();
         $rootScope.$broadcast(Events.filters.tag, getTag(), Events.filters.actionsSelect);
     };
 
+    function setSystemHealth(option) {
+        $location.search('withActions', option.withActions.toString());
+        $location.search('withoutActions', option.withoutActions.toString());
+        MultiButtonService.setState('inventoryWithActions', option.withActions);
+        MultiButtonService.setState('inventoryWithoutActions', option.withoutActions);
+    }
+
     function getTag () {
         return $scope.selected.tag;
     }
 
     function read () {
-        $scope.selected = find($scope.options, {
+        const option = {
             withActions: MultiButtonService.getState('inventoryWithActions'),
             withoutActions: MultiButtonService.getState('inventoryWithoutActions')
-        });
+        };
```

Comment:
get rid of the MultiButtonService stuff and just set $scope.selected to either what is in the URL or a default value. Can probably even move that to the FilterService like we have for most filters.